client/error.css

```diff
@@ -0,0 +1,4 @@
+#error_Message_Div, #remedy_Message_Div{
```

Comment:
Any reason to separate this out into its own css file?

Reply:
Also for ids, I think the recommended approach is `error-message-div` or something to that effect (all lower case, using hyphens)